rts/lra/lra-coordinator-jar/README.md

```diff
@@ -0,0 +1,28 @@
+Enable LRA JWT Security
+=======================
+
+
+Requirements
+------------
+
+* Always name the JWT token **Realm** as: `narayana-lra-jwt`
+
+* Allowed Roles: `{"admin","participant"}`
+
+Before starting LRA-coordinator, we have to first set some System/Environment variable properties, in order to enable LRA-JWT-security layer.
+
+
+To enable security, set the value as "**true**":
+
+	is.security.enabled
```

Comment:
maybe this should be prefixed with `lra.` or similar to avoid assumptions that this is some overall security.

Reply:
Would `is.lra.security.enabled` be more meaningful? Shall I go with this one?

Reply:
I would suggest only `lra.security.enabled` so we can only add `mp.` in case we move it to the spec. Or in SmallRye we use package so it would be here `io.narayana.lra.security.enabled`. I'll leave it to you.

Reply:
Okay, will go with `lra.security.enabled` then.